tests/test_module.py

```diff
@@ -0,0 +1,161 @@
+from pathlib import Path
+from typing import Callable, Dict, Generator
+
+from mailmerge import MailMerge
+import pytest
+from pytest_mock import MockerFixture
+
+from csv2docx import csv2docx
+
+
+@pytest.fixture
+def cleanoutputdir() -> Generator[None, None, None]:
+    default_outpath = Path.cwd() / "output"
+    yield
+    for f in default_outpath.glob("*"):
+        try:
+            f.unlink()
+        except OSError as e:
+            print(f"Error: {f} : {e.strerror}")
+    default_outpath.rmdir()
+
+
+@pytest.fixture
+def mailmerge_docx() -> MailMerge:
+    return MailMerge(Path.cwd() / "tests/data/example.docx")
+
+
+@pytest.fixture
+def options_gen(tmp_path: Path) -> Callable:
+    def _options_gen(**kwargs: str) -> Dict:
+        default = {
+            "template": "tests/data/example.docx",
+            "data": "tests/data/example.csv",
+            "name": "NAME",
+            "path": str(tmp_path.resolve()),
+            "delimiter": ";",
+        }
+
+        # override values if provided
+        for key, value in kwargs.items():
+            if key in default:
+                default[key] = value
+
+        # convert dict to sequential list, add '--' for CLI options (i.e. the keys)
+        return default
+
+    return _options_gen
```

Comment:
Should we pull out these fixtures into a `conftest.py` to reduce what's in the test file?

Reply:
Yes, I like that. Didn't know about `conftest.py` until IDEAFAST!